runtime/src/accounts_db.rs

```diff
@@ -586,16 +586,34 @@ impl AccountsDB {
     }
 
     fn find_storage_candidate(&self, fork_id: Fork) -> Arc<AccountStorageEntry> {
+        let mut create_extra = false;
         let stores = self.storage.read().unwrap();
 
         if let Some(fork_stores) = stores.0.get(&fork_id) {
             if !fork_stores.is_empty() {
+                if fork_stores.len() <= self.min_num_stores {
+                    let mut total_accounts = 0;
+                    for store in fork_stores.values() {
+                        total_accounts += store.count_and_status.read().unwrap().0;
+                    }
+
+                    // Create more stores so that when scanning the storage all CPUs have work
+                    if (total_accounts / 16) >= fork_stores.len() {
```

Comment:
`>=` ?  not sure how it matters much, 16 is kind of arbitrary, might we prefer a comparison with min_num_stores instead?

Reply:
Yea, the >= is so once we hit 16 accounts total, then we create a new store. Another at 32, 48, 64, etc.

Maybe something like `2048 / (32 + min_num_stores)`

Reply:
That would give something like this:

min_num_stores | accounts_rate
-- | --
2 | 60
4 | 57
8 | 51
16 | 43
32 | 32
64 | 21

Reply:
Those numbers are largely arbitrary and probably doesn't make much difference. Can go with just 16 for now.